app/models/report.rb

```diff
@@ -21,15 +22,17 @@ def created_on
   end
 
   def create_mention
-    mentioning_reports.destroy_all
-    @urls = content.scan(%r{http://localhost:3000/reports/\d+})
-    return if @urls.nil?
+    urls = content.scan(%r{http://localhost:3000/reports/\d+})
+    return if urls.empty?
```

Comment:
```suggestion
    destination_ids = content
      .scan(%r{http://localhost:3000/reports/\d+})
      .map { |url| url.split('/').last }
    return if destination_ids.empty?
```

みたいにした方が正規表現とsplitの"/"の関連性がぱっと目に入ってわかりやすいかも

あと、ここでuniqやフィルタリングしちゃうのも一手です。

```suggestion
    destination_ids = content
      .scan(%r{http://localhost:3000/reports/\d+})
      .map { |url| url.split('/').last }
      .uniq
      .reject { |destination_id| id.to_s == destination_id }
    return if destination_ids.empty?
```